### PR TITLE
clean up trainer class lookups

### PR DESCRIPTION
**Pull request by melthelesbian**
more cleanup

### CODE REVIEW

File: include/constants/trainers.h [hunk 1]
```diff
@@ -117,95 +117,95 @@
...
+#define FACILITY_CLASS_PKMN_RANGER_F        0x45
+#define FACILITY_CLASS_MAGMA_LEADER_MAXIE   0x46
+#define FACILITY_CLASS_LASS                 0x47
+#define FACILITY_CLASS_YOUNG_COUPLE         0x48
+#define FACILITY_CLASS_OLD_COUPLE           0x49
+#define FACILITY_CLASS_SIS_AND_BRO          0x4a
+#define FACILITY_CLASS_STEVEN               0x4b
+#define FACILITY_CLASS_SALON_MAIDEN_ANABEL  0x4c
+#define FACILITY_CLASS_DOME_ACE_TUCKER      0x4d
+#define FACILITY_CLASS_RED                  0x4e
```
Comment (Diegoisawesome) [inline include/constants/trainers.h]: Why did you remove the PKMN_TRAINER part in these four but leave it in the WALLY/BRENDAN/MAY ones above?

Comment (melthelesbian) [reply include/constants/trainers.h]: Oversight, fixed.
